apple-pie/index.html

```diff
@@ -4,36 +4,50 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">;
   </head>
   <body>
-    Apple Pie
+    <div class="container">
```

Comment:
You are lacking some structure in this html
You should be using sections, headers, articles to have it more organised.